docs/real-world-examples/ccfraud.md

```diff
@@ -107,3 +107,21 @@ This sample experiment provides multiple models you can try:
 - **SimpleVAE** : a model composed of 2 encoder LSTM layers and 2 decoder LSTM layers that tries to recreate consumed sequence of transactions, the latent space created by encoder is consumed by a linear layer to perform prediction, takes data ordered by time in sequences that overlap each other
 
 To switch between models, please update the `config.yaml` file in `examples/pipelines/ccfraud/`. Look for the field `model_name` in the `training_parameters` section (use `SimpleLinear`, `SimpleLSTM`, or `SimpleVAE`).
+
+## Distributed training
```

Comment:
I would not go into the details on how to create a compute here.

Instead, you could say the pipeline is "distributed training ready", and point to the parameters that need to be changed.

+ add a link to the provisioning guide